gridstatus/tests/test_miso.py

```diff
@@ -252,7 +253,7 @@ def test_get_solar_forecast_historical(self):
         assert df["Publish Time"].dt.date.unique() == pd.to_datetime(past_date).date()
 
     def test_get_solar_forecast_historical_date_range(self):
-        past_date = self.local_today() - pd.Timedelta(days=250)
+        past_date = self.local_today() - pd.Timedelta(days=200)
```

Comment:
These tests were failing because there was no forecast with the publish date '2023-11-06'. I looked through the source data and confirmed this. There were two forecasts with the publish date '2023-11-07'.